go.mod

```diff
@@ -60,13 +61,15 @@ require (
 	k8s.io/api v0.16.7
 	k8s.io/apimachinery v0.16.7
 	k8s.io/client-go v0.16.7
-	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787 // indirect
 )
 
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.3.1-0.20200227195959-4d242818bf55
-
-replace github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200227233006-38f52c9fec82
+replace github.com/docker/docker => github.com/docker/docker v17.12.0-ce-rc1.0.20200310163718-4634ce647cf2+incompatible
 
 replace github.com/jaguilar/vt100 => github.com/tonistiigi/vt100 v0.0.0-20190402012908-ad4c4a574305
 
+// protobuf: corresponds to containerd
+replace github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
+
+replace golang.org/x/sys => golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
```

Comment:
Could you add a comment here to describe why we need the "replace"?

Reply:
done